### PR TITLE
Feat.load.environment.with.decorator

### PR DESCRIPTION
**Pull request by kuyh**
Add Decorator - Load Environment
Rename Environment Properties - ISLAND_ prefix for all properties

### CODE REVIEW

File: src/services/rpc-service.ts [hunk 1]
```diff
@@ -21,13 +21,6 @@ import { AmqpChannelPoolService } from './amqp-channel-pool-service';
 
 export { IRpcResponse, RpcRequest, RpcResponse };
 
-const RPC_EXEC_TIMEOUT_MS = Environments.getIslandRpcExecTimeoutMs();
```
Comment (sshtel) [inline src/services/rpc-service.ts]: If possible, `import { Environments as Env } from '../utils/environments';` &  `Environments.ISLAND_NO_REVIVER`  could be good for code readability!
Just my personal opinion :)

File: src/utils/environments.ts [hunk 2]
```diff
@@ -1,87 +1,200 @@
...
   }
 
-  static getEndpointSessionGroup(): string | undefined {
-    return process.env.ENDPOINT_SESSION_GROUP;
+  constructor() {
+    LoadEnv(this);
   }
 }
+
+export const Environments = new IslandEnvironments();
```
Comment (sshtel) [inline src/utils/environments.ts]: I can see this is to call constructor however I don't know if this is best. any thoughts?
Why don't we use Singleton class?

File: src/services/rpc-service.ts (hunk 1 above)
Comment (kuyh) [reply src/services/rpc-service.ts]: In environments.ts, use env as property decorator
Env seems confusing name :( 

And as you said, too many refactoring issues.
In my opinion, If rename Environments to env is needed,
Create another pull requests is better (later..)

Comment (sshtel) [reply src/services/rpc-service.ts]: yes, you're right.

**Pull request merged**